hack/observability/grafana/kustomization.yaml

```diff
@@ -1,10 +1,15 @@
 resources:
   - ../namespace.yaml
+  # The kube-state-metrics helm chart will reference a configmap with name `kube-state-metrics-crd-config`.
+  # The configMapGenerator below will create the configmap and append a hash suffix calculated from its
+  # content to the name. Kustomize will append the suffix hash to all references in the helm chart, but
+  # only when the helm chart content is referenced in "resources".
+  # This would not work if the helm chart is configured in this file via the "helmCharts" option.
```

Comment:
This is a little bit confusing, why we need to take care of creating a config map for kube state metrics while installing grafana?

Reply:
Should be fixed now. This should be:
```
  # The grafana helm chart will reference a configmap with name `grafana-dashboards`.
```

Already had it locally but didn't push the last commit